README.md

```diff
@@ -1,17 +1,21 @@
-# GitHub Command Line Tool
+# 🦑 ika
 
```

Comment:
I messaged the owner of https://www.npmjs.com/package/ika, asking for the package name. Waiting for a reply.

Reply:
If the reply is "no", I'll rename to something else.